### PR TITLE
citra-qt: disable directory watcher during CIA installation

### PR DESCRIPTION
**Pull request by BreadFish64**
fixes several errors while installing multiple CIAs

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/4284)

### CODE REVIEW

**Comment by zhaowenlan1779** [comment]
> fixes several errors while installing multiple CIAs

Can you elaborate on this?

**Comment by BreadFish64** [comment]
> > fixes several errors while installing multiple CIAs
> 
> Can you elaborate on this?

@Subv was attempting to install multiple small CIAs, and Citra crashed. I reproduced it multiple times and it never occurred in the same place in the code, but it centered around the game directory watcher. Disabling the watcher during installation fixed the crash, and all the error messages during the installation went away. So there's apparently an issue with watching the directory while new subdirectories and files are being created.

File: src/citra_qt/game_list.cpp [hunk 1]
```diff
@@ -315,6 +316,15 @@ void GameList::setFilterVisible(bool visibility) {
     search_field->setVisible(visibility);
 }
 
+void GameList::setDirectoryWatcherEnabled(bool enabled) {
+    if (enabled)
+        connect(watcher, &QFileSystemWatcher::directoryChanged, this,
+                &GameList::RefreshGameDirectory, Qt::UniqueConnection);
+    else
+        disconnect(watcher, &QFileSystemWatcher::directoryChanged, this,
+                   &GameList::RefreshGameDirectory);
```
Comment (zhaowenlan1779) [inline src/citra_qt/game_list.cpp]: multiline `if` blocks need to be wrapped in `{}`

Comment (BreadFish64) [reply src/citra_qt/game_list.cpp]: eh, they're technically one line but ok

**Pull request merged**